handlers/handlers.go

```diff
@@ -4,46 +4,57 @@ import (
     "net/http"
     "encoding/json"
     "fmt"
-    "log"
+    "sync"
+    "github.com/julienschmidt/httprouter"
     "github.com/skrzepto/IPRO-VAULT-SERVER/models"
-    _ "github.com/jinzhu/gorm/dialects/sqlite"
-    "github.com/jinzhu/gorm"
 )
 
 type Impl struct {
-    DB *gorm.DB
+  sd_map map[string]models.SensorData
+  mu sync.Mutex
 }
 
-func (i *Impl) InitDB() {
-    var err error
-    i.DB, err = gorm.Open("sqlite3", "ipro-vault.db")
+func InitGlobal() *Impl {
+    var i Impl
+    i.sd_map = make(map[string]models.SensorData)
+    return &i
+}
+
+
+func (i *Impl) POST_SensorData_ID(rw http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+    decoder := json.NewDecoder(req.Body)
+    var sd models.SensorData
```

Comment:
It actually bothers me that we have to write 'models.' . It is fine for this review but can we create an issue to have a nicer organization in the future? We are not doing MVC and what we have left makes the code ugly.